api/app/signals/apps/reporting/models/export.py

```diff
@@ -33,3 +43,66 @@ class Meta:
     isoweek = models.IntegerField()
     isoyear = models.IntegerField()
     uploaded_file = models.FileField(upload_to='exports/%Y', storage=storage)
+
+
+class CSVExportManager(models.Manager):
+    def _create_rows(self, signals_qs):
+        pass
+
+    def _dump_signals_csv(self, singals_qs, dump_dir, filename):
+        """
+        Write CSV for selected signal instances, only dump current state.
+        """
+        rows = self._create_rows(signals_qs)
+        with open(os.path.join(), 'w') as csv_file:
+            writer = csv_writer(csv_file)
+
+    def create_csv_export(self, basename, export_parameters):
+        # TODO: implement support for areas
+        interval, categories, _ = get_parameters(export_parameters)
```

Comment:
`_` is the `areas` I guess? Why not call it `areas` instead?

Reply:
not implemented yet, `_` to signal it is not used now